docs/src/main/sphinx/security/security-overview.rst

```diff
@@ -0,0 +1,159 @@
+change and verify the results before proceeding.
+
+.. _security-client:
+
+Securing client access to the cluster
+-------------------------------------
+
+Trino *clients* include the Trino :doc:`CLI </installation/cli>`, the
+:doc:`Web UI </admin/web-interface>`, the :doc:`JDBC driver
+</installation/jdbc>`, and `community-provided clients, such as Python or Go
+clients <https://trino.io/resources.html>`_, and any applications using
+these tools.
+
+All access to the Trino cluster is managed by the coordinator. Thus, securing
+access to the cluster means securing access to the coordinator.
+
+There are three aspects to consider:
+
+* :ref:`cl-access-encrypt`: protecting the integrity of client to server
+  communication in transit.
+* :ref:`cl-access-auth`: identifying users and user name management.
+* :ref:`cl-access-control`: validating each user's access rights.
+
+.. _cl-access-encrypt:
+
+Encyryption
+^^^^^^^^^^^
+
+.. comment:
+   The Trino server uses the standard :doc:`HTTPS protocol and TLS encryption
+   <https>`, formerly known as SSL.
+
+The Trino server uses the standard HTTPS protocol and TLS encryption, formerly
+knownn as SSL.
+
+.. _cl-access-auth:
+
+Authentication
+^^^^^^^^^^^^^^
+
+Trino supports several authentication providers. When setting up a new cluster,
+start with simple password file authentication before configuring another
+provider.
+
+* :doc:`Password file authentication <password-file>`
+* :doc:`LDAP authentication <ldap>`
+* :doc:`OAuth 2.0 authentication <oauth2>`
+* :doc:`Salesforce authentication <salesforce>`
+
+.. _user-name-management:
+
+User name management
+""""""""""""""""""""
+
+Trino provides ways to map the user and group names from authentication
+providers to Trino user names.
+
+* :doc:`User mapping <user-mapping>` applies to all authentication systems,
+  and allows for JSON files to specify rules to map complex user names from
+  some systems (``alice@example.com``) to simple user names (``alice``).
+* :doc:`File-based group provider <group-file>` provides a way to assign a set
+  of user names to a group name to ease access control.
+
+.. _cl-access-control:
+
+Authorization and access control
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. comment:
+   Trino and the included enhanced connectors allow you to :doc:`control
+   access <access-control>` to the data exposed by Trino in configured data
+   sources.
+
+Trino's :doc:`default method of access control <built-in-system-access-control>`
+allows all operations for all authenticated users.
+
+To implement access control, use:
+
+* :doc:`File-based system access control <file-system-access-control>`, where
+  you configure JSON files that specify fine-grained user access restrictions at
+  the catalog, schema, or table level.
+* Configure :doc:`Kerboeros authentication <server>` over HTTPS for client
+  access to the coordinator.
+* Configure the Trino :doc:`CLI <cli>` for connecting to a :doc:`Kerberos
```

Comment:
Collapse the Kerberos CLI link into the above bullet

And maybe this bullet can talk about the CLI supporting different authentication mechanisms .. or drop the bullet

Reply:
collapsed. CLI is just another client, so no special treatment. It's only getting a half-bullet because there's a whole page for this subject in Trino docs. Upstream, things are different.